src/main/java/CancelApplication/CancelApplicationPresenter.java

```diff
@@ -0,0 +1,14 @@
+package CancelApplication;
+
+public class CancelApplicationPresenter implements CancelApplicationOutputBoundary {
```

Comment:
Is having both an output boundary and Presenter necessary? Why have these if they return null?

Reply:
Oops, should have left a comment on my code for this. Yes, you're correct that Paul's presenter is actually his output boundary.

His ACTUAL presenter is the file named "ResponseFormatter" in the screens package. So in my code, I haven't actually written anything for my presenter because I'm not sure yet what info the presenter will pass along to the screens. Does that make sense? It's returning null (for now) because my use case is unfinished until I figure out what I want my screen to display.

Reply:
Oh makes sense! Thank you! Is there a difference between having the presenter in the use case package rather than a screen? 

I see your comment earlier. I must have forgotten it when I was reading your code. From my understanding, your implementation would just remove the group from the list, correct me if I am wrong. So maybe return a list of groups without the one they removed (as strings of course) or a message that the group was removed. This would largely depend on how we display the home screen. I really liked your design on our blueprint, it is very clean and clear. If we did it that way, I would think that the groups have already been somehow presented and you are just refreshing the screen. I hope that makes sense. We have not really discussed home screen as we were really focused on our own use cases, but I think it should be useful when implementing your presenter :)

Reply:
> Oh makes sense! Thank you! Is there a difference between having the presenter in the use case package rather than a screen?

Oops, you're right! Forgot to move this to the screens package after I renamed it. Both the Presenter and the Controller should be in the screens package, yeah.

> So maybe return a list of groups without the one they removed (as strings of course) or a message that the group was removed. This would largely depend on how we display the home screen.

Yes, I totally agree. After we figure out the formatting for the home screen and the other "linking" screens that transition between usecases -- I'll definitely have to modify my presenter to make sure the correct info is given to the screen for the other usecases or whoever. I think this is fine for the time being though, at least until we sit and discuss the home screen as a group?

Reply:
Yes totally! We really need to meet up and discuss. I can understand how it can be difficult to decide how to present when we aren't even certain of where or how it will be presented :(. For now, maybe brainstorm and think of all the possibilities🤩.

Reply:
After looking at this discussion, I believe I should also have an output boundary and presenter and will implement it following the changes made here.